packages/smart-accounts/src/light-account/utils.ts

```diff
@@ -198,71 +194,3 @@ export async function getLightAccountImplAddress<
 
   return trim(storage);
 }
-
-/**
- * Get the light account version definition for the given light account and chain
- *
- * @template {LightAccountBase} TAccount
- * @param {LightAccountBase} account the light account to get the version for
- * @param {Chain} chain - the chain to get the version for
- * @returns {Promise<LightAccountVersionConfig>} the light account version definition for the given light account and chain
- */
-export async function getLightAccountVersionForAccount<
```

Comment:
For context: this was created in an ancient PR here: https://github.com/alchemyplatform/aa-sdk/pull/828/files#diff-0bbd548d4d2b59fe196096a4b14f0a2441592ed90f2dd0000fec9f3f84b0d03eL235

And based on my research, it never seems to have been used internally, just exported.

This function gets you the version of an instance of a LightAccount at runtime, which doesn't actually seem useful, since you would typically have that information already based on how it was constructed.

Removing here to minimized exported API surface.